kvmagent/kvmagent/plugins/mevoco.py

```
@@ -775,6 +767,25 @@ def getDhcpEbtableChainName(dhcpIp):
+def get_ebtables_userdata_chain_name(br_name, l3_network_uuid):
+    # Note: In ebtables v1.8 and above, the maximum allowed length for certain string fields
+    # (e.g., --comment, --set-mark, custom chain names) is limited to 28 characters.
+    if is_ebtables_nf_tables():
+        max_chain_name_len = 28
+        prefix = "UD"
+        uuid_len = 8
+        separator_len = 2  # two '-' characters
+        max_br_len = max_chain_name_len - len(prefix) - separator_len - uuid_len
+        return "%s-%s-%s" % (prefix, br_name[-max_br_len:], l3_network_uuid[:uuid_len])
+    else:
+        EBCHAIN_NAME = "USERDATA-%s-%s" % (br_name[-12:], l3_network_uuid[:8])
```

Ruff (0.14.7)

Local variable `EBCHAIN_NAME` is assigned to but never used

Remove assignment to unused variable `EBCHAIN_NAME`

(F841)